### PR TITLE
WIP: Improve base.py documentation

### PR DESCRIPTION
**Pull request by dwf**
See also #540.

### CODE REVIEW

File: blocks/bricks/base.py [hunk 1]
```diff
@@ -822,7 +822,7 @@ def init(self, *args, **kwargs):
 class ApplicationCall(Annotation):
     """A link between the variable tags and bricks.
 
-    The application call can be used to attach to an apply call auxiliary
+    The application call can be used to attach, to an apply call, auxiliary
```
Comment (rizar) [inline blocks/bricks/base.py]: The "to an apply call" seems really obscure to me. Maybe "to attach to the computation graph auxiliary variables..."?

**Comment by bartvm** [comment]
Woops, merge conflicts because of #543

**Comment by rizar** [comment]
@dwf , would like to finish this one?

**Comment by dwf** [comment]
Oops, forgot about this. Will do it soon.

**Comment by rizar** [comment]
Ping

**Pull request closed without merging**